prac_05/word_counter.py

```diff
@@ -0,0 +1,13 @@
+text = input("Text: ")
+words = text.split()
+word_to_count = {}
+keywords = []
+for word in words:
+    try:
+        word_to_count[word] += 1
+    except KeyError:
+        word_to_count[word] = 1
+        keywords.append(word)
```

Comment:
Is this the most efficient solution to sort these words? Sorting them before they're counted may be a better way to approach this since it avoids storing another copy in `keywords`.